examples/bevy/src/main.rs

```diff
@@ -29,15 +29,14 @@ fn main() {
         .build()
         .unwrap();
 
-    // Next, we make an executor, loading the ELF binary.
-    let mut exec = default_executor_from_elf(env, BEVY_GUEST_ELF).unwrap();
+    // Obtain the default prover.
+    let prover = default_prover();
 
-    // Run the executor to produce a session.
-    let session = exec.run().unwrap();
-
-    // Prove the session to produce a receipt.
-    let receipt = session.prove().unwrap();
+    // Produce a receipt by proving the specified ELF binary.
+    let receipt = prover.prove_elf(env, BEVY_GUEST_ELF).unwrap();
 
+    // The prover already runs a verify internally, but this is how other users
+    // would verify the receipt.
```

Comment:
I like that the original version indicates that the `verify` call isn't really necessary here. What about this 3rd option, which makes the redundancy more explicit?
```suggestion
    // The prover already runs a verify internally and so it's redundant to verify
    // again here. However, this is how other users would verify the receipt:
```